vcd/resource_vcd_org_user.go

```diff
@@ -0,0 +1,352 @@
+			},
+			"org": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Organization this user belongs to",
+			},
+			"role": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    false,
+				Description: "Role within the organization",
+			},
+			"password": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: false,
+
+				ConflictsWith: []string{"password_file"},
+				Description: "The user's password. This value is never returned on read. " +
+					`Either "password" or "password_file" must be included on creation.`,
+			},
+			"password_file": &schema.Schema{
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      false,
+				ConflictsWith: []string{"password"},
+				Description: "Name of a file containing the user's password. " +
+					`Either "password_file" or "password" must be included on creation.`,
+			},
+			"description": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    false,
+				Description: "The Org User's description",
+			},
+			"provider_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  govcd.OrgUserProviderIntegrated,
+				ForceNew: false,
+				Description: "Identity provider type for this this user. One of: 'INTEGRATED', 'SAML', 'OAUTH'. " +
+					"When empty, the default value 'INTEGRATED' is used.",
+			},
+			"full_name": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    false,
+				Description: "The Org User's full name",
+			},
+			"email_address": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    false,
+				Description: "The Org User's email address",
+			},
+			"telephone": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    false,
+				Description: "The Org User's telephone",
+			},
+			"instant_messaging": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    false,
+				Description: "The Org User's telephone",
+			},
+			"is_enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				ForceNew:    false,
+				Description: "True if the user is enabled and can log in.",
+			},
+			"is_group_role": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				ForceNew:    false,
+				Description: "True if this user has a group role.",
+			},
+			"is_locked": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: false,
+				Description: "True if the user account has been locked due to too many invalid login attempts. " +
+					"A locked user account can be re-enabled by updating the user with this flag set to false. " +
+					"Only the system can set the value to true.",
```

Comment:
> "A locked user account can be re-enabled by updating the user with this flag set to false. " 

It sounds like in order to re-enable such user one would need to set the flag to **false**...

Reply:
Correct.
Suggestions?

Reply:
No need to invert the logic of the field name. Suggestion could be this (please confirm however whether it is really the way it works):

"If the user account has been locked due to too many invalid login attempts, the value will change to true (only the system can set the value to true). To unlock the user re-set this flag to false."

Reply:
Confirmed. That's the way it works. Changed text accordingly